src/components/ImportForm/ImportForm.tsx

```diff
@@ -36,6 +36,8 @@ const ImportForm: React.FunctionComponent<ImportFormProps> = ({ applicationName
     },
     namespace,
     secret: '',
+    secrets: [],
```

Comment:
Probably need to name this a bit better to distinguish between auth secret and these secrets.

Reply:
updated it to `importSecrets`